.github/scripts/update-tags.sh

```diff
@@ -46,18 +46,21 @@ jq -r '. | keys[]' "$IMAGEJSON" | while read -r CHART; do
     REGISTRY=$(yq e ".${QUERY}.registry" "$VALUES")
     REPOSITORY=$(yq e ".${QUERY}.repository" "$VALUES")
     VERSION=$(yq e ".${QUERY}.tag" "$VALUES")
```

Comment:
These changes allow blank Registries (IE, docker.io). First time we've seen one in the chart.